src/test/java/seedu/pill/command/FindCommandTest.java

```diff
@@ -1,106 +1,53 @@
 package seedu.pill.command;
 
-import seedu.pill.exceptions.PillException;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Test;
-import seedu.pill.util.Item;
+import seedu.pill.exceptions.PillException;
 import seedu.pill.util.ItemMap;
 import seedu.pill.util.Storage;
 
 import java.io.ByteArrayOutputStream;
 import java.io.PrintStream;
 
-public class FindCommandTest{
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class FindCommandTest {
+    private ItemMap itemMap;
+    private Storage storage;
+    private ByteArrayOutputStream outputStream;
+    private PrintStream printStream;
+    private final PrintStream standardOut = System.out;
+
+    @BeforeEach
+    public void setUp() {
+        itemMap = new ItemMap();
+        storage = new Storage();
+        outputStream = new ByteArrayOutputStream();
+        printStream = new PrintStream(outputStream);
+        System.setOut(printStream);
+    }
```

Comment:
Well done with the refactor to make use of BeforeEach and avoiding repeated code